extensions/pcap/CMakeLists.txt

```diff
@@ -19,11 +19,21 @@
 
 find_package(PCAP REQUIRED)
 
+include(FetchContent)
+
+FetchContent_Declare(pcapplusplus
+		URL      https://github.com/seladb/PcapPlusPlus/archive/refs/tags/v21.05.tar.gz

```

Comment:
Why not use GIT_REPOSITORY and GIT_TAG instead of URL and URL_HASH?

Reply:
I prefer keeping the release tarball. It's faster to download and extract than cloning a git repo.

Reply:
Yeah, in this case git clone would use ~130 MB (instead of the tarball which is only ~40 MB extracted), while not providing any clear benefit.

Reply:
Okay.